travharv/config_build.py

```diff
@@ -187,17 +189,17 @@ def _make_path_parts(self, assert_path):
         """
         Make the path parts by splitting the path string on regex expression
         """
-        REGEXP = r"(?:\w+:\w+|<[^>]+>)"
+        REGEXP = r"\s*/\s*(?![^<]*>)"  # TODO - define the regex expression
```

Comment:
still todo or old comment at the end of the line?

Reply:
redundant indeed, deleted